src/main/res/values-bn-rIN/strings.xml

```diff
@@ -0,0 +1,190 @@
+    <string name="intro_title_2">আপনার HDA অ্যাক্সেস করুন</string>
+    <string name="intro_title_3">আপনার ছবিগুলো ব্রাউজ করুন</string>
+    <string name="intro_title_4">আপনার মিউজিক চালান।</string>
+    <string name="intro_title_5">আপনার সিনেমা এবং ভিডিও লাইব্রেরী চালান।</string>
+    <string name="intro_title_6">প্রস্তুত</string>
+    <string name="intro_tv_1">অ্যান্ড্রয়েড টিভির জন্য Amahi কে স্বাগতম</string>
+    <string name="introducing_cast">কাস্ট প্রবর্তন করা হচ্ছে</string>
+    <string name="label_next">পরবর্তী</string>
+    <string name="label_pause">বন্ধ করুন</string>
+    <string name="label_play">চালু করুন</string>
+    <string name="label_previous">পূর্ববর্তী</string>
+    <string name="language_en">ইংলিশ</string>
+    <string name="language_fr">ফ্রেঞ্চ</string>
+    <string name="language_es">স্প্যানিশ</string>
+    <string name="media_route_menu_title">চালু করুন&amp;#8230</string>
+    <string name="menu_audio_files">ফাইল</string>
+    <string name="menu_back">পেছনে যান</string>
+    <string name="menu_delete">মুছুন</string>
+    <string name="menu_forward">ফরওয়ার্ড</string>
+    <string name="menu_navigation_close">নেভিগেশন বন্ধ করুন</string>
+    <string name="menu_navigation_open">নেভিগেশন খুলুন</string>
+    <string name="menu_offline">অফলাইন</string>
+    <string name="menu_open">খোলা</string>
+    <string name="menu_save">ডিভাইসে সংরক্ষণ করুন</string>
+    <string name="menu_search">অনুসন্ধান</string>
+    <string name="menu_settings">সেটিংস</string>
+    <string name="menu_share">শেয়ার</string>
+    <string name="menu_sort">সাজান</string>
+    <string name="menu_video_subtitle">সাবটাইটেল</string>
+    <string name="message_connection_error">সংযোগ ত্রুটি</string>
+    <string name="message_create_and_try_again">ডাউনলোড এর জন্য ফাইলগুলি সন্ধান করুন</string>
+    <string name="message_delete_file_body">এই ক্রিয়াটি পূর্বাবস্থায় ফিরিয়ে আনা সম্ভব না।</string>
+    <string name="message_delete_file_error">ফাইল মোছার সময় কিছু ত্রুটি ঘটছে</string>
+    <string name="message_delete_file_title">মোছার বিষয়টি নিশ্চিত করতে চান?</string>
+    <string name="message_delete_progress">সম্ভব না।</string>
+    <string name="message_duplicate_file_upload">বর্তমান ফাইলটিতেই লিখতে চান?</string>
+    <string name="message_duplicate_file_upload_body">এই ফাইলটি%1$s ইতিমধ্যে বিদ্যমান।\\nআপনি কি উপস্থিত ফাইলটি প্রতিস্থাপন করতে নিশ্চিত?</string>
+    <string name="message_empty_apps">অ্যাপস অনুপস্থিত</string>
+    <string name="message_empty_files">ফাইল অনুপস্থিত</string>
+    <string name="message_empty_recent_files">সাম্প্রতিক ফাইল অনুপস্থিত</string>
+    <string name="message_empty_shares">শেয়ার অনুপস্থিত</string>
+    <string name="message_error_account_exists">অ্যাকাউন্ট ইতিপূর্বে আছে।</string>
+    <string name="message_error_amahi_anywhere_app">The Amahi Anywhere অ্যাপটি আপনার সার্ভার এর কোন জায়গার আয়ত্তাধীন নয়।দয়া করে The Amahi Anywhere অ্যাপটি আপনার নেটওয়ার্কে ইনস্টল, চলমান আর আয়ত্তাধীন তা নিশ্চিত করুন।</string>
```

Comment:
@chirag-jn yes this sentense is translated properly. I didn't changed The Amahi Anywhere because I thought it is a brnad name and It should be stay as it is. Should I aslo translate "The Amahi Anywhere"?

Reply:
"The Amahi Anywhere" isn't a brand per se but "Amahi Anywhere" is

Reply:
Ok understand I will fix this @chirag-jn